action.yml

```
@@ -1,13 +1,19 @@
 name: "release extension"
 description: "Generate a release for a precompiled gh extension"
 inputs:
-  gpg_fingerprint:
-    description: "GPG fingerprint to use for signing releases"
   build_script_override:
     description: "Path to custom build script for producing binaries to upload"
+  draft_release:
+    description: "Create a draft release"
+  github_token:
+    description: "GitHub token to use, defaults to github.token if unspecified"
   go_version:
     description: "The Go version to use for compiling (supports semver spec and ranges)"
     default: "1.18"
+  gpg_fingerprint:
+    description: "GPG fingerprint to use for signing releases"
+  release_tag_override:
```

```suggestion
  release_tag:
```
Not sure that `override` is necessary here, we should indicate that the tag will be automatically determined if this input is not specified.

---

Removed the suffix and described the default behaviour in [8e1e8c2](https://github.com/cli/gh-extension-precompile/pull/37/commits/8e1e8c21c5e04c619f746fab9d199d59faef4ccd)